terragrunt/aws/cloudfront/cloudfront.tf

```diff
@@ -0,0 +1,95 @@
+resource "aws_cloudfront_distribution" "simplify_privacy_app_cf_distribution" {
+  enabled     = true
+  aliases     = [var.domain]
+  price_class = "PriceClass_100"
```

Comment:
This controls how many regions the CloudFront distribution gets replicated to.  The higher the price class, the more regions (and the more expensive):
https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/PriceClass.html

Reply:
So in this case, we're ok at keeping it at 100?

Reply:
Yup, 100 is good for this app